queenbee/simulation/simulation.py

```diff
@@ -3,39 +3,34 @@
 from pydantic import Field, validator, constr
 
 from ..base.basemodel import BaseModel
+from ..base.metadata import MetaData
 from ..recipe import BakedRecipe
-from ..io.inputs.workflow import WorkflowArguments
-from .status import WorkflowStatus
+from ..io.inputs.submission import SubmissionArguments
 
 
-class Workflow(BaseModel):
-    """Queenbee Workflow.
+class SimulationSubmission(BaseModel):
+    """Queenbee Simulation Submission object.
 
-    A Workflow is a Baked Recipe with some arguments that will be used to execute the
-    recipe.
+    A Simulation submission is a Baked Recipe with the arguments that will be used to
+    execute the recipe as a simulation.
     """
-    type: constr(regex='^Workflow$') = 'Workflow'
+    type: constr(regex='^SimulationSubmission$') = 'SimulationSubmission'
 
     recipe: BakedRecipe = Field(
```

Comment:
I am very tempted to just remove this key, especially if we are going for a pollination centric design. Here is my thinking:

1. Local Run: Queenbee Luigi pre-compiles the baked recipe so it only needs inputs and metadata such as the recipe name, the name of the simulation etc...
2. Cloud Run: Queenbee Argo needs to compile the argo workflow using both baked recipe and arguments but this is not a user facing operation and arguably might be specific to queenbee-argo. In this sense I can make it work using 2 different objects rather than one combined object.

If we follow this approach I would also get rid of the `SimulationSubmissionInfo` object in favour of this one.

Alternatively if we think that the `pollination` centric stuff should exclusively live in the `queenbee-pollination` code base then we could have this object only contain `arguments` and deffer to `queenbee-pollination` to provide the `name`, `keywords` and `recipe` information.

Reply:
Good thinking! We will remove this and will add new fields for `source`, `name` and `keywords`.